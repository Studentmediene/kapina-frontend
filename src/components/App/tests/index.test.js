import React from 'react';
import { shallow } from 'enzyme';

import { App } from '../';

describe('<App />', () => {
  it('renders correctly', () => {
    const props = {
      routes: [],
      pathname: '/path',
    };

    const tree = shallow(<App {...props} />);
    expect(tree).toMatchSnapshot();
  });

  it('renders plainpost correctly', () => {
    const props = {
      routes: [],
      pathname: '/plainpost',
    };

    const tree = shallow(<App {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
