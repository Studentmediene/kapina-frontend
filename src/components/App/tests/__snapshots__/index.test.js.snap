// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="container"
>
  <Header />
  <div
    className="content"
  >
    <Switch />
  </div>
  <Connect(Footer) />
  <Connect(Player) />
</div>
`;

exports[`<App /> renders plainpost correctly 1`] = `
<div
  className="container"
>
  <div
    className="content"
  >
    <Switch />
  </div>
</div>
`;
